src/components/TextCard/TextCard.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { HTMLAttributes } from 'react';
+import { cnb } from 'cnbuilder';
+import { AnimateInView, AnimationType } from '../Animate';
+import { Text } from '../Typography';
+import { FlexBox } from '../FlexBox';
+import { bgTextColorPairsBlackWhite, BgTextColorPairBlackWhiteType } from '../../utilities/datasource';
+import * as styles from './TextCard.styles';
+
+export type TextCardProps = HTMLAttributes<HTMLDivElement> & {
+  text?: string;
+  color?: BgTextColorPairBlackWhiteType;
+  font?: 'serif' | 'druk6' | 'druk7';
+  xsColSpan?: styles.ColSpanType;
+  smColSpan?: styles.ColSpanType;
+  mdColSpan?: styles.ColSpanType;
+  lgColSpan?: styles.ColSpanType;
+  xlColSpan?: styles.ColSpanType;
+  xxlColSpan?: styles.ColSpanType;
+  animation?: AnimationType;
+  delay?: number;
+};
```

Comment:
This TextCard is kind of a grid filler card. I allow for it to span various number of columns at different breakpoints (1-3 are the only options currently) to act as fillers for a grid